third_party/terraform/tests/resource_app_engine_standard_app_version_test.go

```diff
@@ -84,6 +84,8 @@ resource "google_app_engine_standard_app_version" "foo" {
     }
   }
 
+  inbound_services = ["INBOUND_SERVICE_WARMUP"]
```

Comment:
Can we test having multiple values in this? (not in a separate test, just put like 3ish values in this array)